qiskit/extensions/standard/h.py

```diff
@@ -16,14 +16,14 @@
 Hadamard gate.
 """
 import numpy
-
 from qiskit.circuit import Gate
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit import QuantumRegister
+from qiskit.circuit.controlledgate import ControlledGate
 from qiskit.qasm import pi
-from qiskit.extensions.standard.u2 import U2Gate
 
 
+# pylint: disable=cyclic-import
```

Comment:
Is the `pylint-ignore` still needed here?

Reply:
yes